catalog/ui/catalog-ui-search/src/main/webapp/js/cql.js

```diff
@@ -122,6 +125,12 @@ define([
             'AFTER': temporalClass,
             'DURING': temporalClass
         },
+
+        // TODO: for extra credit, these could be figured out while building the syntax tree
```

Comment:
I did things generically so additional filter functions we want to support won't need any more work in this module, save for this map. I figured adding an entry here is minor compared to other work involved in supporting new methods. Maybe wait and see if a new filter function comes up with variable params (which will require this work)?